### DISCUSSION TITLE
Update about the project

### DISCUSSION THREAD

**Discussion by dphang** [serverless-nextjs/serverless-next.js#2439] [Announcements]
Hey all!

Hope things are going well. I have an important update to share...

First, I know there are many open issues that have been overlooked and I apologize for not giving them as much attention as I had last year. Unfortunately I am the only core maintainer and I have had very limited time recently and also haven't had the need to use this actively as I have temporarily stopped using this for my side projects. Plus, having to maintain both the deployment logic and keep up with new Next.js features and emulate the logic, has been pretty hard. I had reached out a few times to hopefully get a core maintainer to help with some things, but it's been tough.

I'm glad that the community has grown over the past years but unfortunately it's also increasingly complex to maintain... so I am making a final call for any contributors who would like to participate!

I will of course try to continue looking at PRs and also fix some of the major bugs/regressions, but long term for the health of this project, I need to figure out how to make things simpler to maintain (e.g using as much next.js server code as possible, so we don't need to emulate it).

Thanks,
Daniel

**Comment by coltenkrauter** [comment]
Thanks for this update. I have a few questions for you.

Has anything changed since your last update? 
Do you foresee any ability to maintain this project going forward? 
What would it take for you or Vercel to help maintain this?
If a core maintainer stepped up, would you work with them or would this be completely theirs to bare?

**Comment by coltenkrauter** [comment]
For anyone following this discussion, looks like there may be new alternatives.

https://open-next.js.org/
https://github.com/serverless-nextjs/serverless-next.js/issues/2497#issuecomment-1350814284
